CHANGELOG.md

```diff
@@ -26,8 +34,8 @@ Operators may need to tune their kernel configuration to allow a higher number o
 
 ## Performance
 
-- **M3DB**: Use a single results object for merging postings lists across blocks, rather than creating a result per block and discarding them immediately thereafter (#1474)
-- **M3DB**: Improvement to applying back pressure on writes with a full commitlog, which should improve recovery when dealing with spiky traffic (#1482)
+- **M3DB**: Use a single results object for merging postings lists across blocks, rather than creating a result per block and discarding them immediately thereafter. This should reduce memory utilization significantly for queries that look back over a large number of blocks (#1474)
```

Comment:
Meant to update 0.7.3?

Reply:
Yeah, i missed some comments left on the 0.7.3 changelog unfortunately